src/server/server_family.cc

```diff
@@ -888,6 +888,13 @@ GenericError RewriteConfigFile() {
   return {};
 }
 
+bool IsMaster() {
+  if (!ServerState::tlocal()) {
```

Comment:
maybe add a comment that mentioning the corner case you saw?
so that next time someone won't try to remove it?